ksp_plugin_adapter/ksp_plugin_adapter.cs

```diff
@@ -69,6 +70,14 @@ public partial class PrincipiaPluginAdapter : ScenarioModule {
 
   private Krakensbane krakensbane_;
   private NavBall nav_ball_;
+  private UnityEngine.Texture compass_nav_ball_texture_;
```

Comment:
KSP's conventions seem to tend towards nav_ball, since it's navBall or NavBall, but then again they're KSP conventions. Going with nav_ball, Navball as discussed.